### ISSUE TITLE
Error when sharing a project with SVN

### ISSUE THREAD

**Issue by marcgise** [subclipse/subclipse#151]
I am getting an error when sharing a project with SVN. The error is the following:
```
An error has occurred. See error log for more details.
Cannot invoke "org.tigris.subversion.subclipse.core.resources.SVNWorkspaceRoot.getLocalRoot()" because the return value of "org.tigris.subversion.subclipse.core.resources.LocalResource.getWorkspaceRoot()" is null
```
![image](https://user-images.githubusercontent.com/4680243/102350701-96609d80-3fa5-11eb-8b0c-2569a2d73753.png)

These are the plugins and versions I have installed:
![image](https://user-images.githubusercontent.com/4680243/102350762-b42e0280-3fa5-11eb-9332-9b69394bed11.png)
![image](https://user-images.githubusercontent.com/4680243/102350789-bf812e00-3fa5-11eb-9de4-42773f9c81d0.png)

Eclipse version is 2020-09 (4.17.0), running with OpenJDK 15

**Comment by selsemore** [comment]
Do you get this for any project that you try to share? Is there a stacktrace in the error log details?

**Comment by markphip** [comment]
Can you install and try the latest build by adding this update site?  https://dl.bintray.com/subclipse/snapshots/

It fixed the problem for me but I got a different NPE than you did so not sure if it fixes your problem.

**Comment by marcgise** [comment]
Now it seems to work, thanks

**Comment by markphip** [comment]
Thanks for the followup, I have published the 4.3.3 release with the fix.

**Comment by PyvesB** [comment]
I was getting flooded by hundreds of errors when interacting with the Package Explorer:

```
Error Decorating null
java.lang.NullPointerException
	at org.tigris.subversion.subclipse.core.resources.LocalResource.hasSymlinkParent(LocalResource.java:202)
	at org.tigris.subversion.subclipse.core.resources.LocalResource.isIgnored(LocalResource.java:123)
	at org.tigris.subversion.subclipse.core.resources.LocalFolder.members(LocalFolder.java:116)
	at org.tigris.subversion.subclipse.core.resources.LocalFolder.isDirty(LocalFolder.java:162)
	at org.tigris.subversion.subclipse.ui.decorator.SVNLightweightDecorator.decorate(SVNLightweightDecorator.java:385)
	at org.eclipse.ui.internal.decorators.LightweightDecoratorDefinition.decorate(LightweightDecoratorDefinition.java:247)
	at org.eclipse.ui.internal.decorators.LightweightDecoratorManager$LightweightRunnable.run(LightweightDecoratorManager.java:105)
	at org.eclipse.core.runtime.SafeRunner.run(SafeRunner.java:45)
	at org.eclipse.ui.internal.decorators.LightweightDecoratorManager.decorate(LightweightDecoratorManager.java:360)
	at org.eclipse.ui.internal.decorators.LightweightDecoratorManager.getDecorations(LightweightDecoratorManager.java:346)
	at org.eclipse.ui.internal.decorators.DecorationScheduler$1.ensureResultCached(DecorationScheduler.java:386)
	at org.eclipse.ui.internal.decorators.DecorationScheduler$1.run(DecorationScheduler.java:362)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
```

The problem has gone away with the latest release, thanks! :+1:
